<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPackageName</class>
 <widget class="QDialog" name="SetPackageName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwPackages">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Package Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePackageName">
      <property name="placeholderText">
       <string>example: com.rovio.angrybirds</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnListPkgOne">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>List All Packages</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnListPkgTwo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>(permission required)</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbDoubleClickInstruction">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Double click&lt;/span&gt; the desired package name, then click&lt;span style=&quot; font-weight:600;&quot;&gt; Next&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbError">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>Error output</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
